pkg/cloud/services/networking/securitygroups.go

```diff
@@ -546,6 +548,17 @@ func (s *Service) createSecurityGroupIfNotExists(openStackCluster *infrav1.OpenS
 			record.Warnf(openStackCluster, "FailedCreateSecurityGroup", "Failed to create security group %s: %v", groupName, err)
 			return err
 		}
+
+		if len(openStackCluster.Spec.Tags) > 0 {
+			mc := metrics.NewMetricPrometheusContext("security_group", "update")
```

Comment:
we use [ReplaceAllAttributesTags](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/pkg/cloud/services/networking/client.go#L110) function in networking package so that we do not need to care metrics every time.

Reply:
should have seen that. Updated it. Thanks